HW13/Common/WebElements/MyWebElement.cs

```diff
@@ -0,0 +1,112 @@
+﻿using HW13.Common.Drivers;
+using HW13.Common.Extensions;
+using OpenQA.Selenium;
+using System.Collections.ObjectModel;
+using System.Drawing;
+
+namespace HW13.Common.WebElements
+{
+    public class MyWebElement : IWebElement
+    {
+        protected By By { get; set; }
+
+        protected IWebElement WebElement => WebDriverFactory.Driver.GetWebElementWhenExist(By);
+
+        public string TagName => WebElement.TagName;
+
+        public string Text => WebElement.Text;
+
+        public bool Enabled => WebElement.Enabled;
+
+        public bool Selected => WebElement.Selected;
+
+        public Point Location => WebElement.Location;
+
+        public Size Size => WebElement.Size;
+
+        public bool Displayed => WebElement.Displayed;
+
+        public IWebElement WrappedElement => throw new NotImplementedException();
+
+        public MyWebElement (By by)
+        {
+            By = by;
+        }
+
+        public void Clear() => WebElement.Clear();
+
+        public void Click()
+        {
+            try
+            {
+                WebElement.Click();
+            }
+            catch (ElementClickInterceptedException)
+            {
+                ScrollIntoView();
+                WebElement.Click();
+            }
+        }
+
+        public void ClickAndWait()
+        {
+            try
+            {
+                WebElement.Click();
+            }
+            catch (ElementClickInterceptedException)
+            {
+                ScrollIntoView();
+                WebElement.Click();
+            }
+            Thread.Sleep(500);
+        }
+
+        public void RightClick()
```

Comment:
If your method contains the only line, so you convert your method to lambda expression. Fix it everywhere